src/software/ai/hl/stp/play/kickoff_enemy_play_test.cpp

```diff
@@ -33,20 +33,15 @@ TEST_F(KickoffEnemyPlayTest, test_kickoff_enemy_play)
 
     std::vector<ValidationFunction> terminating_validation_functions = {
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            // TODO: Fix bug with robot three not shadowing the enemy kicker in
-            // kickoff_enemy_play
-            // https://github.com/UBC-Thunderbots/Software/issues/1945
```

Comment:
@nimazareian I saw that you were assigned to this ticket and it seems like I was able to pass the test when I uncomment the lines below. Just wanted to give you a heads up.